### PR TITLE
George

### PR DESCRIPTION
**Pull request by georgeb79**
SEER test cases updated

### CODE REVIEW

File: src/main/java/com/nci/automation/web/CommonUtils.java
Comment (matakevin) [inline src/main/java/com/nci/automation/web/CommonUtils.java:1]: Line 430 camel casing seems to be off on "lastNAmeRandomizer"

File: src/test/java/ServiceNow/SEER/Constants/Registration_Constants.java
Comment (matakevin) [inline src/test/java/ServiceNow/SEER/Constants/Registration_Constants.java:1]: I could be wrong but to Addresses normally have underscores and hyphens?

File: src/test/java/ServiceNow/SEER/Features/LandingPage.feature
Comment (matakevin) [inline src/test/java/ServiceNow/SEER/Features/LandingPage.feature:1]: Double annotation "@@" on line 3 shouldn't be there

File: src/main/java/com/nci/automation/web/CommonUtils.java
Comment (georgeb79) [reply src/main/java/com/nci/automation/web/CommonUtils.java:1]: Corrected. Thanks!

File: src/test/java/ServiceNow/SEER/Constants/Registration_Constants.java
Comment (matakevin) [reply src/test/java/ServiceNow/SEER/Constants/Registration_Constants.java:1]: Got it, great!  Thanks for clarifying!

**Review by matakevin — APPROVED** [review]
LGTM

**Review by shomiruddin94 — APPROVED** [review]

File: src/test/java/ServiceNow/SEER/Features/SEERDataAccessRequestPage.feature
Comment (matakevin) [inline src/test/java/ServiceNow/SEER/Features/SEERDataAccessRequestPage.feature:1]: Your regression tag on line 3 is misspelled

**Review by matakevin — CHANGES_REQUESTED** [review]
Hey George, I only have one change that I found that should be changed.  On the SEERDataAccessRequestPage.feature file

**Review by matakevin — APPROVED** [review]
LGTM

File: src/test/java/ServiceNow/SEER/Features/SEERDataAccessRequestPage.feature
Comment (iamdez99) [reply src/test/java/ServiceNow/SEER/Features/SEERDataAccessRequestPage.feature:1]: verified

**Pull request merged**